scripts/run_ch_sampler.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash                                                                                                                            
+#                                                                                                                                      
+#SBATCH --job-name=mc_ch                                                                                                               
+#SBATCH --output=mc_ch.out                                                                                                             
+#SBATCH --mail-type=END,FAIL                                                                                                           
+#SBATCH --nodes=1                                                                                                                      
+#SBATCH --ntasks-per-node=8                                                                                                            
+#SBATCH --gres=gpu:1                                                                                                                   
+#SBATCH -C k40                                                                                                                         
+#SBATCH --time=12:00:00                                                                                                                
+#SBATCH --mem=32gb                                                                                                                     
+
+top_dir=/home/adegennaro
```

Comment:
I wouldn't add things that are specific to only you as a user. Could just use $HOME ?

Reply:
done